mobile/src/main/java/org/openhab/habdroid/model/OpenHABNotification.java

```diff
@@ -48,14 +52,14 @@ public static OpenHABNotification fromJson(JSONObject jsonObject) throws JSONExc
             try {
                 created = format.parse(jsonObject.getString("created")).getTime();
             } catch (ParseException e) {
-                // keep created at null
+                // keep created at 0
             }
         }
 
         return new AutoValue_OpenHABNotification.Builder()
                 .icon(jsonObject.optString("icon", null))
                 .severity(jsonObject.optString("severity", null))
-                .message(jsonObject.optString("message", null))
+                .message(jsonObject.getString("message"))
```

Comment:
What happens now, when no message is present?

Reply:
A `JSONException` will be thrown ... IOW, message is required now. A notification without message doesn't make sense anyway, though.

Reply:
Maybe it's better to default to `""`. It doesn't make sense, but the app doesn't crash when the server sends invalid messages.

Reply:
The JSONException is catched, thus the app won't crash. That's the very point of this PR ;-) Currently, if message is missing, the Builder object throws a NPE.